sigle/src/modules/layout/components/AppHeader.tsx

```diff
@@ -0,0 +1,74 @@
+import { EyeOpenIcon as EyeOpenIconBase } from '@radix-ui/react-icons';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { styled } from '../../../stitches.config';
+import { Button as ButtonBase, Container, Flex, Text } from '../../../ui';
+import { useAuth } from '../../auth/AuthContext';
+import { Logo } from './Logo';
+
+const Header = styled('header', Container, {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  mt: '$10',
+  mb: '$20',
```

Comment:
Regarding the header, do we have the same bottom spacing in all the pages? If not might be better to have the same mt and mb value here and adjust individually on the different pages.